deploy/examples/create-external-cluster-resources.py

```diff
@@ -969,6 +973,16 @@ def get_rbd_node_caps_and_entity(self):
 
         return caps, entity
 
+    def get_healthchecker_caps_and_entity(self):
+        entity = "client.healthchecker"
+        caps = {
+            "mon": "allow r, allow command quorum_status, allow command version",
+            "mgr": "allow command config",
+            "osd": f"profile rbd-read-only, allow rwx pool={self._arg_parser.rgw_pool_prefix}.rgw.meta, allow r pool=.rgw.root, allow rw pool={self._arg_parser.rgw_pool_prefix}.rgw.control, allow rx pool={self._arg_parser.rgw_pool_prefix}.rgw.log,allow x pool={self._arg_parser.rgw_pool_prefix}.rgw.buckets.index",
```

Comment:
Set this value in unit test `{self._arg_parser.rgw_pool_prefix}`

Reply:
It is an optional flag, so add an if on top of it either the flag value or the default value 
`DEFAULT_RGW_POOL_PREFIX`

Reply:
Can we check fir if else here `{self._arg_parser.rgw_pool_prefix}`

Reply:
actually we're already setting the value but it was nor applying for upgrade so added for that also